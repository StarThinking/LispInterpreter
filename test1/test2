(S . (T . (W . (Z . NIL))))
((A . (B . NIL)) . (C . NIL))
(A . (B . ((C . (D . NIL)) . NIL)))
((A . NIL) . NIL)
(A . ((B . C) . NIL))
