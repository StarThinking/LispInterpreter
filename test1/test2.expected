(S T W Z)
((A B) C)
(A B (C D))
((A))
(A . ((B . C) . NIL))
